MACOS.md

```diff
@@ -198,14 +184,63 @@ brew upgrade openssl     || brew install openssl
 ```
 
 
-## Sublime Text 3 - Your text editor
+## Visual Studio Code
+
+### Installation
+
+1. Download [Visual Studio Code for macOS](https://code.visualstudio.com/Download).
```

Comment:
Why not go with a simple:

```bash
brew install --cask visual-studio-code
```

Seems it even natively detect `M1`: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/visual-studio-code.rb#L4-L10

Then you can have students launch VS Code for the first time with:

```bash
'/Applications/Visual Studio Code.app'
```

and then explain that we can put a nice shortcut with the command palette!

Reply:
That's better! Thank your for the command :pray: